Alkiiro/refactor-tractor
# Second inheritance repo method and testing created

What's this PR do?
This PR update created the second method in our parent class Repo.  The method is `getUserDailyStats` and removed the repetitive nature of the different method names in Sleep,Activity, and Hydration.  The testing for each class has been completed in the repo testing file.  Index.js has also been updated for all the changes.

Where should the reviewer start?
`getDailyStats` was changed throughout the Activity class and Activity test file.  `TotalOzDay` was updated through the Hydration class and Hydration test file.   `getDailySleepHours` was updated throughout Sleep.

How should this be manually tested?
I would pull the repo down and console.log to make sure data is being pulled correctly. All tests passed on my end.

Any background context you want to provide?
This Repo class will continue to be built out to provide more methods for the repetitive methods in child classes.
